Assets/Scripts/Models/Furniture.cs

```diff
@@ -244,28 +291,53 @@ public List<string> ReplaceableFurniture
         }
     }
 
-    // This is a multipler. So a value of "2" here, means you move twice as slowly (i.e. at half speed)
-    // Tile types and other environmental effects may be combined.
-    // For example, a "rough" tile (cost of 2) with a table (cost of 3) that is on fire (cost of 3)
-    // would have a total movement cost of (2+3+3 = 8), so you'd move through this tile at 1/8th normal speed.
-    // SPECIAL: If movementCost = 0, then this tile is impassible. (e.g. a wall).
+    /// <summary>
+    /// Gets the movement cost multiplier that this furniture has. 1 is default, 2 is twice as slow.
+    /// Tile types and environmental effects will be combined with this value (additive).
+    /// If this value is '0' then the furniture is impassable.
+    /// </summary>
+    /// <value>The movement cost multiplier the furniture has.</value>
     public float MovementCost { get; private set; }
 
+    /// <summary>
+    /// Gets a value indicating whether the furniture can close a room (e.g. act as a wall).
+    /// </summary>
     public bool RoomEnclosure { get; private set; }
 
-    // For example, a sofa might be 3x2 (actual graphics only appear to cover the 3x1 area, but the extra row is for leg room.)
+    /// <summary>
+    /// Gets the width of the furniture.
+    /// </summary>
     public int Width { get; private set; }
 
+    /// <summary>
+    /// Gets the height of the furniture.
+    /// </summary>
     public int Height { get; private set; }
 
+    /// <summary>
+    /// Gets the code used for Localization of the furniture.
+    /// </summary>
     public string LocalizationCode { get; private set; }
 
+    /// <summary>
+    /// Gets the description of the furniture if there is no localization support.
```

Comment:
Here is `UnlocalizedDescription` for door from xml `<UnlocalizedDescription>furn_door_desc</UnlocalizedDescription>`